src/api/middleware/deps-injection.js

```diff
@@ -0,0 +1,6 @@
+const i18n = require('../../locales');
+
+module.exports = async function depsInjectionMiddleware(ctx, next) {
+  ctx.t = i18n.t.bind(i18n);
+  await next();
```

Comment:
```suggestion
```

Reply:
this `next` is required right? (since it's a middleware)
but it should be `return next()` I think

Reply:
Middleware can either return a promise, or call `next`. In my case, I shouldn't have removed it.